Makefile

```diff
@@ -87,9 +72,7 @@ test-skopeo:
 	@echo === Testing skopeo build
 	@export GOPATH=$$(mktemp -d) && \
 		skopeo_path=$${GOPATH}/src/github.com/containers/skopeo && \
-		vendor_path=$${skopeo_path}/vendor/github.com/containers/image && \
 		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
-		rm -rf $${vendor_path} && cp -r . $${vendor_path} && rm -rf $${vendor_path}/vendor && \
```

Comment:
This probably needs to be updated, but it must not be just removed; the goal is to run the `skopeo` tests against the containers/image version being tested.

Reply:
Reverted :)

Reply:
(If I’m reading this correctly, the final `rm -rf $${vendor_path}/vendor` could be removed, but it also does not hurt leaving it in, and it could be beneficial in case someone ran `go mod vendor`.)

Reply:
Let's keep it in just in case :)